projects/begin-wallet-core.md

```diff
@@ -0,0 +1,86 @@
+# Project Name
+Begin Wallet Core
+
+# Preferred Maturity Level
+Growth -
+We're in production using this library on our wallet and looking for more active support from the Foundation or TAC mentorship to reach bigger goals.
```

Comment:
What we see in this repository is a library or a fully functional wallet (white labeled)?

Reply:
For now, it is just the library that can facilitate the creation of a wallet.